simulators/README.md

```diff
@@ -54,6 +58,10 @@ git clone git@github.softwareag.com:IOTA/oee-simulators.git
 cd oee-simulators/simulators
 docker build -t oee-simulators .
 docker save -o image.tar oee-simulators
+```
+In [cumulocity.json](oee-simulators/simulators/cumulocity.json), change "version" from "@project.version@" to version number you want in format xx.xx.xx (example: "version": "12.20.11")
```

Comment:
please add another example: "1.0.latest"
this syntax allows to use the same version for multiple uploads; otherwise you always get an error when you try to upload the same version again...